### ISSUE TITLE
Same feed?

### ISSUE THREAD

**Issue by Posandu** [forem/forem#16105]
**Describe the bug**

Same feed on the homepage

**To Reproduce**

Go to the homepage and refresh it, it's the same posts list

**Expected behaviour**

Post list should be updated

**Screenshots**

First load
![image](https://user-images.githubusercontent.com/76736580/149511456-2e7f976c-95c2-4394-8ad8-66b9a79ec164.png)
After refreshing
![image](https://user-images.githubusercontent.com/76736580/149511493-ee3f603d-6271-4c59-b2eb-abd73c44c6db.png)

**Desktop (please complete the following information):**

- OS, version: 10
- Browser, version: Latest Chrome

**Comment by jeremyf** [comment]
@Posandu this is the expected behavior of the latest feed algorithm changes.  See https://forem.dev/foremteam/feed-test-experiment-2-results-new-query-strategy-has-pulled-even-j07.

tl;dr - the previous feed algorithm was a bit like drawing cards from a deck.  The deck was a subset of all articles recently posted.  Each time you'd go to the feed page you'd in essence shuffle roughly the same deck and draw a handful of cards.  The new algorithm creates a more stable feed (e.g., it doesn't shuffle the cards but instead picks cards from all articles that should be more relevant to you).

(Paging @benhalpern)
